AlphaWallet/Tokens/Views/TextField.swift

```diff
@@ -7,12 +7,16 @@ protocol TextFieldDelegate: AnyObject {
     func doneButtonTapped(for textField: TextField)
     func nextButtonTapped(for textField: TextField)
     func shouldChangeCharacters(inRange range: NSRange, replacementString string: String, for textField: TextField) -> Bool
+    func didBeginEditing(in textField: TextField)
 }
 
 extension TextFieldDelegate {
     func shouldChangeCharacters(inRange range: NSRange, replacementString string: String, for textField: TextField) -> Bool {
         return true
     }
+    func didBeginEditing(in textField: TextField) {
+        return
```

Comment:
This default implementation for the protocol function is so that we don't edit other files right? Could you help to add a TODO to this function or better still a follow up TODO in this PR's comment so we'll remember to create an issue to follow up to remove it? eg.

- [ ] Do xxx

Reply:
This default implementation is to avoid editing other files. I don't understand what you want. What are we removing?

Reply:
A reminder to remove the default implementation

Reply:
What if I made this an optional method?